tests/cert.tests.ps1

```diff
@@ -0,0 +1,481 @@
+                [bool] $useLiveData = $false
+            )
+
+            $flag = $false
+            if ($useLiveData) {
+                $index = 0
+                # Loop through each item in the output.
+                foreach ($item in $output) {
+                    if ($user -and $type) {
+                        if ($item.'System' -match $server -and $item.'User' -match $user -and $item.'Type' -match $type) {
+                            $flag = $true
+                            break
+                        }
+                    } else {
+                        if ($user) {
+                            # If the system matches the server and user, break the loop.
+                            if ($item.'System' -match $server -and $item.'User' -match $user) {
+                                $flag = $true
+                                break
+                            }
+                        } elseif ($type) {
+                            if ($item.'System' -match $server -and $item.'Type' -match $type) {
+                                $flag = $true
+                                break
+                            }
+                        } else {
+                            # If the system matches the server, break the loop.
+                            if ($item.'System' -match $server) {
+                                $flag = $true
+                                break
+                            }
+                        }
+                    }
+                    # Increment the index by 1.
+                    $index = $index + 1
+                }
+            } else {
+                $index = $output.'Index'
+            }
+            if (-Not $flag) {
+                Write-LogToFile -Type ERROR -message "$server or $user is not matching in the $output"
+            } else {
+                return $index
+            }
+        }
+
+        Start-SetupLogFile
+        $inputData = Get-Content -Raw 'InputData.json' | ConvertFrom-Json
+        $server = $inputData.'SDDC Manager'
+        $sddcManagerUser = $inputData.'User'
+        $sddcManagerPass = $inputData.'Password'
+        $certAuthorityFqdn = $inputData.'certAuthorityFqdn'
+        $certAuthorityUser = $inputData.'certAuthorityUser'
+        $certAuthorityPass = $inputData.'certAuthorityPass'
+        $certAuthorityTemplate = $inputData.'certAuthorityTemplate'
+        $commonName = $inputData.'commonName'
+        $organization = $inputData.'organization'
+        $organizationUnit = $inputData.'organizationUnit'
+        $locality = $inputData.'locality'
+        $state = $inputData.'state'
+        $country = $inputData.'country'
+        $outputDirectory = $inputData.'outputDirectory'
+        $email = $inputData.'email'
+        $keysize = $inputData.'keysize'
+        $domain = $inputData.'Domains'[0]
+        $esxiServer = $inputData.$domain.'ESXi Hosts'[0]
+        $cluster = $inputData.$domain.'Clusters'[0]
```

Comment:
```suggestion
        #$esxiServer = $inputData.$domain.'ESXi Hosts'[0]
        #$cluster = $inputData.$domain.'Clusters'[0]
```
since these are not used should be comment out for now

Reply:
Best to just remove instead of comment.

```suggestion
```

Reply:
@Ashwini-Ravi addressed this in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/commit/400b75366f04d6c47d1edc5acbd905f9219adead.